background.ts

```diff
@@ -938,19 +938,19 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
       return ['no changes necessary', ''];
     }
 
-    const imageAllowListConf = '/usr/local/openresty/nginx/conf/image-allow-list.conf';
+    const allowedImagesConf = '/usr/local/openresty/nginx/conf/allowed-images.conf';
     const rcService = k8smanager.backend === 'wsl' ? 'wsl-service' : 'rc-service';
 
     // Update image allow list patterns, just in case the backend doesn't need restarting
     // TODO: review why this block is needed at all
-    if (cfg.containerEngine.imageAllowList.enabled) {
-      const allowListConf = BackendHelper.createImageAllowListConf(cfg.containerEngine.imageAllowList);
+    if (cfg.containerEngine.allowedImages.enabled) {
+      const allowListConf = BackendHelper.createImageAllowListConf(cfg.containerEngine.allowedImages);
 
-      await k8smanager.executor.writeFile(imageAllowListConf, allowListConf, 0o644);
+      await k8smanager.executor.writeFile(allowedImagesConf, allowListConf, 0o644);
```

Comment:
Do we also need to delete the old file, if it exists? (I guess it's not actually used anymore, so maybe it's fine?)

Reply:
I was doing that, but Jan talked me out of it because leaving it there doesn't do any harm, and the less code we write the better.

Reply:
The discussion was actually about deleting `/usr/local/openresty/nginx/conf/nginx.conf`, `/usr/local/openresty/nginx/conf/resolver.conf`, and `/etc/logrotate.d/openresty` when openresty is not enabled.

But I agree that it also applied to the old file; it is less than 1KB and doesn't hurt anybody.

It is better to have no code, than code that performs something not really necessary because having no code has likely fewer bugs. 😄